src/main/java/frc/robot/RobotContainer.java

```diff
@@ -24,6 +25,7 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private SN_Extreme3DStick coDriverStick = new SN_Extreme3DStick(RobotMap.ControllerMap.CO_DRIVER_STICK);
+  private SN_DualActionStick DriverStick = new SN_DualActionStick(RobotMap.ControllerMap.DRIVER_STICK);
```

Comment:
We will want to use this in our command, so does anything needs to change to make that work?